client/app/templates/components/project-milestone.hbs

```diff
@@ -68,133 +71,131 @@
-            {{#each dedupedVotes as |vote|}}
-              {{#if vote.dispHasRequiredField}}
-                <li class="grid-x small-margin-bottom">
-                  <div class="cell shrink small-margin-right">
-                    <span data-test-rec-symbol="{{vote.disposition.id}}">
-                      <RecommendationResultIcon
-                        @recommendation={{vote.disposition.recommendationLabel}}
-                      />
-                    </span>
-                  </div>
-                  <div class="cell auto">
-                    <strong data-test-rec-label="{{vote.disposition.id}}">{{vote.disposition.recommendationLabel}}</strong>
-                    <small class="display-inline-block" data-test-vote-date="{{vote.disposition.id}}">
-                      Vote:&nbsp;{{~moment-format vote.disposition.dcpDatereceived "LL"~}}
-                    </small>
-                    <div class="text-tiny">
-                      {{#each vote.voteActions as |action index|}}
-                        {{#if action.dcpName}}
-                          <span class="label light-gray tiny-margin-bottom tiny-margin-right" data-test-vote-actions-list="{{vote.disposition.id}}{{index}}">
-                            {{action.dcpName}}
-                            <small>{{action.dcpUlurpnumber}}</small>
-                          </span>
-                        {{/if}}
-                      {{/each}}
+                    <div class="cell auto">
+                      <strong data-test-rec-label="{{vote.disposition.id}}">{{vote.disposition.recommendationLabel}}</strong>
+                      <small class="display-inline-block" data-test-vote-date="{{vote.disposition.id}}">
+                        Vote:&nbsp;{{~moment-format vote.disposition.dcpDatereceived "LL"~}}
+                      </small>
+                      <div class="text-tiny">
+                        {{#each vote.voteActions as |action index|}}
+                          {{#if action.dcpName}}
+                            <span class="label light-gray tiny-margin-bottom tiny-margin-right" data-test-vote-actions-list="{{vote.disposition.id}}{{index}}">
+                              {{action.dcpName}}
+                              <small>{{action.dcpUlurpnumber}}</small>
+                            </span>
+                          {{/if}}
+                        {{/each}}
+                      </div>
                     </div>
-                  </div>
-                </li>
-              {{/if}}
-            {{/each}}
-          {{/borough-pres-deduped-votes-list}}
-        </ul>
-      {{/if}}
-    {{/each}}
-  {{/hearings-list-for-milestones-list}}
-</div>
-
-{{yield}}
+                  </li>
+                {{/if}}
+              {{/each}}
+            {{/borough-pres-deduped-votes-list}}
+          </ul>
+        {{/if}}
+      {{/each}}
+    {{/hearings-list-for-milestones-list}}
+  {{/if}}
```

Comment:
Aline mentioned that this other info (outcomes, links) should never show if the milestone is "not started" / until the milestone is "in progress" or "completed"